vip-helpers/vip-utils.php

```diff
@@ -1098,15 +1098,22 @@ function is_automattician( $user_id = false ) {
 		return false;
 	}
 
-	// $vip_support = WPCOM_VIP_Support_User::init();
-
-	if ( WPCOM_VIP_Support_User::is_verified_automattician( $user->ID ) ) {
-		return true;
+	if ( A8C_PROXIED_REQUEST && WPCOM_VIP_Support_User::is_a8c_email( $user->user_email ) ||
```

Comment:
Mixing an 'automattician' check and a proxied 'bypass' here seems a little messy. What's the benefit to `is_automattician()` returning true if you're not 'verified', but are proxied?

I think if we want to take the proxy status into account, we should be using `is_proxied_automattician()`, and `is_automattician()` strictly checks if this user is really an Automattician. As is, this introduces some variability into what actually constitutes an Automattician to the system, which is harder to grok. IMO they should be separate concerns unless we require you to be proxied for `is_automattician()` to return `true` in all cases, which I don't think we want.